### PR TITLE
Node install

### PR DESCRIPTION
**Pull request by gz**

### CODE REVIEW

**Comment by gz** [comment]
ping @lalithsuresh

File: deploy/Dockerfile [hunk 1]
```diff
@@ -4,29 +4,37 @@
...
-    locale-gen
+  locale-gen
 ENV LC_ALL en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal
-# web-console build tools
-RUN curl -fsSL https://deb.nodesource.com/setup_19.x | bash -
-RUN apt-get install --yes nodejs
+# web-console build tools:
```
Comment (lalithsuresh) [inline deploy/Dockerfile:30]: Do the web-console build tools need to be part of the base image? Note that the compiler-server derives from the base image, and I don't think it needs all this in it.

Comment (gz) [reply deploy/Dockerfile:30]: I don't know but I just change the currently deprecated installation method. The node installation happened here previously. If we need to move it lets do one-by-one I can open a 2nd PR for it?

Comment (gz) [reply deploy/Dockerfile:30]: If I understand correctly you suggest to move this here? https://github.com/feldera/feldera/blob/main/deploy/Dockerfile#L51

Comment (lalithsuresh) [reply deploy/Dockerfile:30]: Yes it can be a 2nd PR. And yes, moving it to the point you indicated should work.

**Pull request merged**